<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="pCheckBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>记住账号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="pLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pLineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="aComboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>128</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="aLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>43</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pCheckBox</zorder>
  <zorder>lButton</zorder>
  <zorder>pLabel</zorder>
  <zorder>pLineEdit</zorder>
  <zorder>aComboBox</zorder>
  <zorder>aLabel</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
